astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
Is this the best solution?
(upper, lower order is confusing).

Wouldn't this be better (None meaning no selection, allowing one-sided cuts):
```python
AirmassRange.__init__(lower_limit=None, upper_limit=None)
```

In any case, please update the docstring to reflect the current parameters.
And I'm +1 to add some more explanation on this one and an `Examples` section that explains common usage.
Probably when we see the examples and explanation it'll also be easier to decide what the best solution for this is.